demisto_sdk/commands/common/content/objects/pack_objects/pack.py

```diff
@@ -288,20 +297,50 @@ def sign_pack(self, logger: logging.Logger, dumped_pack_dir: Path, sign_director
         except Exception as error:
             logger.error(f'Error while trying to sign pack {self.path.name}.\n {error}')
 
+    def is_server_version_ge(self, client, server_version_to_check):
+        server_version = get_demisto_version(client)
+        return LooseVersion(server_version.base_version) >= LooseVersion(server_version_to_check)  # type: ignore
+
     def upload(self, logger: logging.Logger, client: demisto_client):
         """
         Upload the pack zip to demisto_client
         Args:
             logger (logging.Logger): System logger already initialized.
             client: The demisto_client object of the desired XSOAR machine to upload to.
-
         Returns:
             The result of the upload command from demisto_client
         """
-
+        if self.is_server_version_ge(client, '6.5.0'):
+            try:
+                logger.info('Uploading...')
+                return client.upload_content_packs(file=self.path, skip_verify='true')  # type: ignore
+
+            except Exception as err:
```

Comment:
it will be nice if a few words about skip verify and server version check will be in the doc string

Reply:
done